migration_original/ODS1Stage/tables/Base/ProviderToDisplaySpecialty/spu_translated_ProviderToDisplaySpecialty.sql

```diff
@@ -0,0 +1,169 @@
+CREATE OR REPLACE PROCEDURE ODS1_STAGE.BASE.SP_LOAD_PROVIDERTODISPLAYSPECIALTY(IsProviderDeltaProcessing BOOLEAN) -- Parameters
+    RETURNS STRING
+    LANGUAGE SQL
+    EXECUTE AS CALLER
+    AS  
+DECLARE 
+---------------------------------------------------------
+--------------- 0. Table dependencies -------------------
+---------------------------------------------------------
+    
+-- Base.ProviderToDisplaySpecialty depends on:
+--- Raw.VW_PROVIDER_PROFILE
+--- Base.Provider
+--- Base.ProviderToSpecialty
+--- Base.DisplaySpecialtyRule
+--- Base.DisplaySpecialtyRuleToSpecialty
+--- Base.ProviderToCertificationSpecialty
+--- Base.DisplaySpecialtyRuleToCertificationSpecialty
+--- Base.ProviderToClinicalFocus
+--- Base.DisplaySpecialtyRuleToClinicalFocus
+
+---------------------------------------------------------
+--------------- 1. Declaring variables ------------------
+---------------------------------------------------------
+
+    delete_statement STRING;
+    truncate_statement STRING;
+    select_statement STRING; -- CTE and Select statement for the insert
+    insert_statement STRING; -- Insert statement 
+    status STRING; -- Status monitoring
+   
+---------------------------------------------------------
+--------------- 2.Conditionals if any -------------------
+---------------------------------------------------------   
+   
+BEGIN
+    IF (IsProviderDeltaProcessing) THEN
+           delete_statement := 'DELETE FROM Base.ProviderToDisplaySpecialty 
+                                    WHERE ProviderID IN 
+                                        (SELECT ProviderID 
+                                         FROM Raw.ProviderDeltaProcessing)';
+
+           select_statement := '
+           WITH CTE_ProviderBatch AS (
+                SELECT ProviderId
+                FROM Raw.ProviderDeltaProcessing
+            ),';
+    ELSE
+           truncate_statement := 'TRUNCATE TABLE Base.ProviderToDisplaySpecialty';
+           
+           select_statement := '
+           WITH CTE_ProviderBatch AS (
+                SELECT ProviderId
+                FROM Base.Provider),';
+    END IF;
+
+
+---------------------------------------------------------
+----------------- 3. SQL Statements ---------------------
+---------------------------------------------------------     
+
+--- Select Statement
+
+-- If conditionals:
+select_statement := select_statement || 
+                    $$ CTE_ProviderBatch AS (
+                                SELECT ProviderId
+                                FROM Base.Provider),
+                                
+                            NotExists AS (
```

Comment:
add cte_ to NotExists
```suggestion
                            CTE_NotExists AS (
```